src/modules/account/RadixTransferAccountSystem.ts

```diff
@@ -21,12 +23,24 @@ import Decimal from 'decimal.js';
 import { RadixTokenDefinition } from '../token/RadixTokenDefinition';
 import { RadixAtomStatusIsInsert, RadixAtomObservation } from '../..';
 
+enum AtomOperation {
```

Comment:
Is TransferAccountSystem the right place for this enum?

Reply:
I would say no ... but it is removed in the next PR anyway

Reply:
There's a duplicate in types.ts though

Reply:
Yes, it will also be removed.

Reply:
But yeah it doesn't make much sense to commit it like this I guess, even though it will change ...

Reply:
Fixed.